client-sdk/spice-java-sdk-sample/README.md

```diff
@@ -0,0 +1,170 @@
+# Spice with Java sdk sample
+
+<https://github.com/spiceai/spice-java>
+
+## Start spice runtime
+
+```shell
+spice run
```

Comment:
Let's reference the quickstart you're using here.

Reply:
Updated - thank you